src/main/java/bio/terra/service/filedata/flight/ingest/IngestPopulateFileStateFromFileStep.java

```diff
@@ -24,22 +25,23 @@
 // Populate the files to be loaded from the incoming array
 public class IngestPopulateFileStateFromFileStep implements Step {
     private final LoadService loadService;
-    private final ObjectMapper objectMapper;
     private final int maxBadLines;
     private final int batchSize;
 
     public IngestPopulateFileStateFromFileStep(LoadService loadService,
-                                               ObjectMapper objectMapper,
                                                int maxBadLines,
                                                int batchSize) {
         this.loadService = loadService;
-        this.objectMapper = objectMapper;
         this.maxBadLines = maxBadLines;
         this.batchSize = batchSize;
     }
 
     @Override
     public StepResult doStep(FlightContext context) {
+        // Ensure that file ingestion works with extra key-value pairs
+        ObjectMapper objectMapper = new ObjectMapper()
+            .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
I think you should add setting
```
.registerModule(new Jdk8Module())
.registerModule(new JavaTimeModule())
```
to the ObjectMapper. I think that will process the Instant type better.